src/swagger-specs/query-service.yaml

```diff
@@ -803,6 +1087,18 @@ parameters:
     required: true
     type: string
     in: header
+  x-sandbox-id:
+    name: x-sandbox-id
+    description: 'The sandbox ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the documentation covering making API calls in the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).'
```

Comment:
This is not true...where did you get this info? Sandboxes are managed via the Platform UI or Sandbox API, not from Platform integrations in Dev Console.

Have you confirmed that the user needs to provide this header in addition to `x-sandbox-name`, and this isn't just an internal requirement? Normally we just ignore this one.